ndmg/stats/qa_reg.py

```diff
@@ -37,6 +37,7 @@
 mpl.use("Agg")  # very important above pyplot import
 from nilearn.plotting.edge_detect import _edge_map as edge_map
 import matplotlib.pyplot as plt
+from ndmg.utils.qa_utils import get_min_max, opaque_colorscale,pad_im
```

Comment:
minor: add correct spacing on these imports, looks weird right now

Reply:
fixed it in the latest version

Reply:
https://github.com/neurodata/ndmg/pull/366/commits/29c0e0c3750f095a833aa6ff25e9d31ae5e353c7